dlp/api/Snippets.Tests/DeidentifyDataStoredInCloudStorageTests.cs

```diff
@@ -0,0 +1,230 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License
+
+using Google.Api.Gax.ResourceNames;
+using Google.Cloud.BigQuery.V2;
+using Google.Cloud.Dlp.V2;
+using Google.Cloud.Storage.V1;
+using System.IO;
+using System;
+using Xunit;
+using Google.Apis.Bigquery.v2.Data;
+
+namespace GoogleCloudSamples
+{
+    public class DeidentifyDataStoredInCloudStorageTests : IClassFixture<DlpTestFixture>
+    {
+        private DlpTestFixture _fixture;
+        public DeidentifyDataStoredInCloudStorageTests(DlpTestFixture fixture) => _fixture = fixture;
+
+        internal static void CreateDatasetTable(string datasetId, string tableId, BigQueryClient bigQueryClient)
+        {
+            var dataset = new Dataset
+            {
+                Location = "US"
+            };
+
+            // Create the dataset.
+            var datasetResponse = bigQueryClient.CreateDataset(datasetId, dataset);
+            var schema = new TableSchemaBuilder
+            {
+                { "Name", BigQueryDbType.String },
+                { "Email", BigQueryDbType.String },
+            }.Build();
+
+            // Create the table.
+            datasetResponse.CreateTable(tableId, schema: schema);
+
+            // Insert the rows inside the table which we have created above.
+            BigQueryInsertRow[] rows = new BigQueryInsertRow[]
+            {
+                new BigQueryInsertRow(insertId: "row1") {
+                    { "Name", "James" },
+                    { "Email", "james@gmail.com" }
+                },
+                new BigQueryInsertRow(insertId: "row2") {
+                    { "Name", "Gary" },
+                    { "Email", "gary@gmail.com" }
+                }
+            };
```

Comment:
Copy paste?
The is sample uses BQ as an output to write the details of transformations done.

Reply:
It was my bad. Removed the addition part.